docs/content/guide/user-activities.md

```diff
@@ -0,0 +1,7 @@
+## User Activity Logs
+
+User Activity module gives you access to control each passport identities' activity logs. You can filter actrivity logs by identity, action type, module, date or even document id. In a standart build of Spica, you can see all user activies without any time limitation.
+
+> IMPORTANT: Because of no time limit on user acitivity logs, we suggest you to clear user activities in a time interval. Otherwise you should consider your server hardwares to use Spica instance with hight performance.
```

Comment:
```suggestion
> HINT: Because of no time limit on user activity logs, we suggest you clear user activities in a time interval. To accomplish this, you can create a function that clears the activity collection on a regular basis by using the **Schedule** trigger.
```